platform/default/include/mbgl/storage/offline_database.hpp

```diff
@@ -150,7 +149,35 @@ class OfflineDatabase : private util::noncopyable {
 
     optional<uint64_t> offlineMapboxTileCount;
 
-    bool evict(uint64_t neededFreeSize);
+    class DatabaseSizeChangeStats {
+    public:
+        explicit DatabaseSizeChangeStats(OfflineDatabase*);
```

Comment:
nit: reference instead of pointer?

Reply:
I need it copyable :disappointed: https://github.com/mapbox/mapbox-gl-native/pull/15622/files#diff-5b457ebb155db345aa36bd628436e111R315